backend/routes/amendBooking/index.ts

```diff
@@ -31,14 +34,19 @@ export default function createRoutes({ bookingService, availabilityCheckService,
     asyncMiddleware(changeVideoLink.submit())
   )
 
-  router.get('/video-link-not-available/:bookingId', asyncMiddleware(videoLinkNotAvailable.view()))
-  router.post('/video-link-not-available/:bookingId', asyncMiddleware(videoLinkNotAvailable.submit()))
+  router.get('/video-link-not-available/:bookingId', checkUpdatePresent, asyncMiddleware(videoLinkNotAvailable.view()))
+  router.post(
+    '/video-link-not-available/:bookingId',
+    checkUpdatePresent,
+    asyncMiddleware(videoLinkNotAvailable.submit())
+  )
 
   router.get('/room-no-longer-available/:bookingId', asyncMiddleware(roomNoLongerAvailable.view()))
```

Comment:
probably want to wrap this too? maybe we aren't in create and need to?

Reply:
We're not currently wrapping it in create but can add that as well

Reply:
Sorted in second commit